cmd/oapi-codegen/oapi-codegen.go

```diff
@@ -30,6 +30,7 @@ import (
 
 func errExit(format string, args ...interface{}) {
 	_, _ = fmt.Fprintf(os.Stderr, format, args...)
+	fmt.Println("")
```

Comment:
Possibly, we could do:

```suggestion
	_, _ = fmt.Fprintf(os.Stderr, format + "\n", args...)
```

Then it's definitely going to include the newline, or we make sure that anyone calling this method adds the `\n`, which is probably my preference

Reply:
Yes. Curious to know why do you prefer caller appending the newline. Wouldn't it result in more unterminated error messages (of course, the patch could result in double-terminated messages if the callers don't know, but it could be solved by appending an 'ln' to the function name, like in Println)?

Reply:
Looking at the code calling it, yep, there are quite a few places where we're not explicitly adding the `\n` at the end, so agreed we need to do something about it.

I think keeping it as `errExit` probably is fine, although I like the use of `ln`.

Maybe we can add some logic that if it doesn't end with a `\n` we append it, then users of the method can specify it explicitly, or it can be added automagically, so we don't end up with duplicates?

Reply:
Implemented the automagic addition of newline since checking the last char won't be resource intensive or error-prone in Go compared to C. Please check commit f0460847c0d9417423b4ae64f4fc36825cdf7ba4 .